docs/reference/cluster/update-settings.asciidoc

```diff
@@ -46,42 +53,51 @@ value. Here is an example:
 
 [source,js]
 --------------------------------------------------
-curl -XPUT localhost:9200/_cluster/settings -d '{
+PUT /_cluster/settings
+{
     "transient" : {
-        "discovery.zen.minimum_master_nodes" : null
+        "indices.recovery.max_bytes_per_sec" : null
     }
-}'
+}
 --------------------------------------------------
+// CONSOLE
+// TEST
 
 Reset settings will not be included in the cluster response. So
 the response for the last example will be:
 
 [source,js]
 --------------------------------------------------
 {
+    ...
     "persistent" : {},
     "transient" : {}
 }
 --------------------------------------------------
+// CONSOLE
+// TESTRESPONSE[s/\.\.\./"acknowledged": true,/]
```

Comment:
I don't think it adds any value, and actually distracts from what the example is trying to point out. I tend to think anything not pertaining to what the example is explaining should not be displayed.